modules/project-docs/pages/compatibility.adoc

```diff
@@ -80,6 +80,36 @@ The following JDK releases are supported:
 Please make sure you run on one of the latest patch releases, since they provide stability improvements and security fixes in general.
 
 
+== OS Compatibility
+
+In general, the JVM removes worries about underlying OS compatibility, 
```

Comment:
Should we rephrase this sentence 
_JVM removes worries about to_ 
**JVM eliminates concerns about**